apps/console/src/features/applications/models/application.ts

```diff
@@ -799,3 +799,8 @@ export interface idpInfoTypeInterface {
     name: string;
     redirectTo?: string;
 }
+
+export enum SubjectTypes {
```

Comment:
Let's add a small comment to describe what this enum is.

Reply:
added with [a5f0e39](https://github.com/wso2/identity-apps/pull/4284/commits/a5f0e399c3a9fa90a48a937face8fe7d2f554faf)